hal-x86_64/src/interrupt/mod.rs

```diff
@@ -69,7 +69,7 @@ impl Handlers<crate::X64> for TestHandlersImpl {
     {
         let mut vga = vga::writer();
         vga.set_color(vga::ColorSpec::new(vga::Color::Red, vga::Color::Black));
-        writeln!(&mut vga, "code fault\n{:#?}", cx.registers()).unwrap();
+        log::error!("code fault\n{:#?}", cx.registers());
         vga.set_color(vga::ColorSpec::new(vga::Color::Green, vga::Color::Black));
```

Comment:
Same deal with removing `set_color` calls.

Reply:
yeah, i did that in my follow-up PR that makes the interrupt handlers more cross-platform... probably should've done it here instead